### ISSUE TITLE
Provide details description for AWS Integration in web

### ISSUE THREAD

**Issue by stevenGravy** [gravitational/teleport#46827]
Expected behavior:

All integrations including AWS would have a details message

Current behavior:

GitLab, Okta have a details message but the AWS integration is blank.

<img width="1343" alt="image" src="https://github.com/user-attachments/assets/a8ae1ccc-1b7d-434d-a46f-b5c763f65db4">

<img width="934" alt="image" src="https://github.com/user-attachments/assets/d2d9a9a9-a3d6-469f-8242-348ae3d4c4ad">

Bug details:
- Teleport version: 16.4.0
- Recreation steps

1. Create a AWS integration and a Okta
2. Only the Okta will include details text

**Comment by zmb3** [comment]
While we're there, can we add logos for Okta and Gitlab like we already have for AWS?

**Comment by zmb3** [comment]
@kimlisa is this still an issue? I was looking at the conversation in #51235 and it seems like maybe you were going to fix it?

**Comment by kimlisa** [comment]
yes, it was added here https://github.com/gravitational/teleport/pull/51539/files#diff-39d6ead4c09146a78b2dd1fb13ab281b908f85034c6de9eaddfcef6cd53a7ecfR464
